.github/workflows/build-publish.yml

```diff
@@ -23,7 +23,7 @@ jobs:
         fetch-depth: 0
     - name: Go Build
       run: |
-        echo "machine github.com login machine-parsable password ${{ secrets.GH_PAT_MACHINE_PARSABLE }}" > ~/.netrc
+        echo "machine github.com login machine-parsable password ${{ secrets.GH_PAT_CLASSIC_MACHINE_PARSABLE }}" > ~/.netrc
```

Comment:
i don't think you need this change.  That was something I was testing earlier and forgot to remove.  can you try without this change?

Reply:
Sure, I just changed it back to the way it was.